packages/inter-protocol/test/reserve/setup.js

```diff
@@ -14,51 +23,35 @@ const faucetRoot = './test/vaultFactory/faucet.js';
  */
 
 /**
- * NOTE: called separately by each test so AMM/zoe/priceAuthority don't interfere
+ * NOTE: called separately by each test so zoe/priceAuthority don't interfere
  *
  * @param {*} t
  * @param {ManualTimer | undefined} timer
  * @param {FarZoeKit} farZoeKit
- * @param {Issuer} runIssuer
- * @param {{ committeeName: string, committeeSize: number}} electorateTerms
  */
-const setupReserveBootstrap = async (
-  t,
-  timer,
-  farZoeKit,
-  runIssuer,
-  electorateTerms,
-) => {
-  const runBrand = await E(runIssuer).getBrand();
-  const centralR = { issuer: runIssuer, brand: runBrand };
-
-  const ammSpaces = await setupAmmServices(
-    t,
-    electorateTerms,
-    centralR,
-    timer,
-    farZoeKit,
-  );
-  const { produce } =
+const setupReserveBootstrap = async (t, timer, farZoeKit) => {
```

Comment:
This is the sort of test bootstrap setup that we could perhaps do away with using the swingset bootstrap testing approach @turadg presented today.